gradle.properties

```diff
@@ -2,6 +2,6 @@ projectName=dataframe
 version=0.11.0
 jupyterApiTCRepo=
 kotlin.jupyter.add.scanner=false
-org.gradle.jvmargs=-Xmx2G
+org.gradle.jvmargs=-Xmx4G
```

Comment:
back to 4 hey?

Reply:
Yes, korro needs it, otherwise fails with OOM

Reply:
nice